extensions/amp-carousel/0.1/scrollable-carousel.js

```diff
@@ -190,7 +190,6 @@ export class AmpScrollableCarousel extends BaseCarousel {
    */
   scrollHandler_() {
     const currentScrollLeft = this.container_./*OK*/scrollLeft;
-    this.pos_ = currentScrollLeft;
```

Comment:
Where is this updated now? Or do we no longer need it completely?

Reply:
We don't need it at all. https://github.com/ampproject/amphtml/pull/17507/files#diff-2acff622566bdbcedb28a9434a237d66R232 is already updating it later in a more generic way that is applicable to navigate bu button and scroll.